### DISCUSSION TITLE
Is there a way to run GeneActivity without Fragment.tsv file?

### DISCUSSION THREAD

**Discussion by myylee** [stuart-lab/signac#1003] [Q&A] [answered]
Hi there,

Thanks for the great package. 

I'm just wondering if there is a way to run GeneActivity() without the Fragment.tsv file? Is it possible to recreate a Fragment.tsv-like file using the cell-peak count matrix? 

Thanks!

Michelle

**Comment by timoast** [comment, marked as answer]
It's not possible, see https://github.com/timoast/signac/issues/957

**Reply by myylee** [reply]
Read the thread, made sense. Thanks!
